### DISCUSSION TITLE
Where is cyberduck's log file kept?

### DISCUSSION THREAD

**Discussion by rbelew** [iterate-ch/cyberduck#14355] [Q&A] [answered]
[v8.5.6 on OSX 10.15.7]

I'm getting "Transfer incomplete" messages and trying to figure out why.

An earlier version had a "Toggle log file" option, and I recall there used to also be a preference setting showing where the log file lived?  I've looked in my user's `Library/Application Support/Cyberduck` folder.

**Comment by AliveDevil** [comment, marked as answer]
The log files are located in `~/Library/Logs/Cyberduck`.

**Reply by AliveDevil** [reply]
The option used to be called "Toggle log drawer" in "View", which is deprecated.
You can reach the log file through Preferences -> Connection -> Reveal.

**Reply by dkocher** [reply]
Additionally the transcript that could previously be monitored in the log drawer is now streamed to the unified logging system on macOS. You can stream it in _Terminal.app_ using 

```
log stream --predicate '(process == "Cyberduck") && (category == "transcript")' --level info
```

**Comment by dkocher** [comment]
Documentation can be found in [Logging Output](https://docs.cyberduck.io/cyberduck/support/#logging-output).

**Comment by rbelew** [comment]
i can't mark both @AliveDevil  and @dkocher responses as THE answer but I wish I could!  I missed it in the doc, but am grateful for the pointers to the raw log file, too. thank you all.
